### PR TITLE
Show autosave indicator

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description
image

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-pkg/src/components/AppTopBar.vue [hunk 1]
```diff
@@ -43,11 +43,15 @@
...
           </template>
+          <span v-if="hasAutosave" class="oc-flex oc-flex-middle">
+            <oc-icon name="refresh" color="white" v-oc-tooltip="autoSaveTooltipText" />
+          </span>
           <template v-if="mainActions.length && resource">
             <context-action-menu
               :menu-sections="[
                 {
                   name: 'main-actions',
+
```
Comment (kulmann) [inline packages/web-pkg/src/components/AppTopBar.vue:54]: why the new line?

File: packages/web-pkg/src/components/AppTopBar.vue [hunk 2]
```diff
@@ -114,20 +120,33 @@ export default defineComponent({
...
+    },
     resource: {
       type: Object as PropType<Resource>,
       default: null
     }
   },
   emits: ['close'],
-  setup(props) {
-    const { $gettext } = useGettext()
+  setup: function (props) {
```
Comment (kulmann) [inline packages/web-pkg/src/components/AppTopBar.vue:133]: why the change?

Comment (AlexAndBear) [reply packages/web-pkg/src/components/AppTopBar.vue:133]: IDE AUTO FORMATOR -.-

File: packages/web-pkg/src/components/AppTopBar.vue [hunk 3]
```diff
@@ -43,6 +43,9 @@
               />
             </oc-drop>
           </template>
+          <span v-if="hasAutosave" class="oc-flex oc-flex-middle">
+            <oc-icon name="refresh" color="white" v-oc-tooltip="autoSaveTooltipText" />
```
Comment (kulmann) [inline packages/web-pkg/src/components/AppTopBar.vue:47]: maybe we have some magic css variable for the color. if you find something good, if not then leave it with `white`. ;-)

**Review by kulmann — APPROVED** [review]
💯
